crates/rust-analyzer/src/lsp_ext.rs

```diff
@@ -337,3 +337,31 @@ pub struct CommandLink {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub tooltip: Option<String>,
 }
+
+pub enum OpenDocs {}
+
+impl Request for OpenDocs {
+    type Params = OpenDocsParams;
+    type Result = DocumentationLink;
+    const METHOD: &'static str = "rust-analyzer/openDocs";
```

Comment:
This seems to be a generally useful thing, not specific to rust, so let's use `experimental/externalDocs` name

Reply:
I really wish commands were standardized because I think this would be better as a command.